example/simple_webcam.py

```diff
@@ -0,0 +1,27 @@
+from pyacq import create_manager, ImageViewer
+from pyqtgraph.Qt import QtCore, QtGui
+
+
+man = create_manager()
+
+# this create the dev in a separate process (NodeGroup)
+nodegroup = man.create_nodegroup()
+dev = nodegroup.create_node('WebCamAV', name = 'cam0')
+dev.configure(camera_num = 0)
+dev.output.configure(protocol = 'tcp', interface = '127.0.0.1', transfertmode = 'plaindata')
+dev.initialize()
+
+#view is a Node in local QApp
+app = QtGui.QApplication([])
+
+viewer = ImageViewer()
+viewer.configure()
+viewer.input.connect(dev.output)
```

Comment:
There is still 2 problems in that design:
  * the manager do not know this connection
  * the node do not known the node.name and output.name whit wich it is connected. (easy to fix).

Someting to add in a new PR.